packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -100,15 +109,30 @@ export const iconMetadata: {
     IconBookMagnifyingGlass: {
         tags: ['analyze']
     },
-    IconCalendar: {
-        tags: ['schedule']
-    },
     IconCalendarCheckLines: {
         tags: ['test', 'plan', 'schedule']
     },
+    IconCalendarClock: {
+        tags: ['time', 'schedule']
+    },
     IconCalendarDay: {
         tags: ['date', 'time', 'schedule']
     },
+    IconCalendarDays: {
+        tags: ['schedule']
+    },
+    IconCalendarLines: {
+        tags: ['agenda']
+    },
+    IconCalendarRectangle: {
+        tags: []
+    },
```

Comment:
Is there and intended metaphor for this icon's usage? From the initial issue with name `calendar-filled-rectangle-year_16x16` I'd guess "year" but I also don't know how well this icon represents a "year". If anything this looks like a "month" picker based on the other visuals for day and week that cover parts of the calendar.

Reply:
Yes, I guess it should list the metaphor "year", but I agree that it seems ambiguous. @NIbokeefe can you confirm? Year? Month? Both?

Reply:
Only barely related but I came across this calendar icon set that people were raving about on the internet. They also had to stretch to extend the metaphor to "year".

https://www.reddit.com/r/DesignPorn/comments/1azciu3/microsoft_to_do_repeat_icons/

![image](https://github.com/user-attachments/assets/a210b899-df97-436b-a866-27499d61f31c)

Reply:
> Yes, I guess it should list the metaphor "year", but I agree that it seems ambiguous. @NIbokeefe can you confirm? Year? Month? Both?
![Uploading Screenshot 2024-07-16 at 10.17.55 AM.png…]()

Reply:
> Yes, I guess it should list the metaphor "year", but I agree that it seems ambiguous. @NIbokeefe can you confirm? Year? Month? Both?
![Uploading Screenshot 2024-07-16 at 10.17.55 AM.png…]()

Reply:
<img width="305" alt="Screenshot 2024-07-16 at 10 23 13 AM" src="https://github.com/user-attachments/assets/91fd032c-8e64-43a7-9587-59531aa8249b">

Reply:
I agree, I didn't like this one for year either but they are so small it wasn't possible at add 365 or anything like that.  It was Madhamitas design and I attempted a couple other versions. Went over them with Fred and let him be the 'D' which one.

<img width="337" alt="Screenshot 2024-07-16 at 10 11 13 AM" src="https://github.com/user-attachments/assets/1cb75ebc-aeae-4d64-91f0-79209f84153e">

Reply:
I'm updating the metaphors for `IconCalendarDay` (removing "time" and "schedule", so now just "date"), `IconCalendarDays` (adding "month" alongside existing "schedule"), and `IconCalendarRectangle` (added "year"). I am not adding tags for words that are already in the name of the icon (e.g. "day" or "week").